packages/opentelemetry-core/src/platform/browser/sdk-info.ts

```diff
@@ -15,11 +15,12 @@
  */
 
 import { VERSION } from '../../version';
+import { TelemetrySdkLanguageValues } from '@opentelemetry/semantic-conventions';
 
 /** Constants describing the SDK in use */
 export const SDK_INFO = {
```

Comment:
should we use semantic conventions for  name/runtime like https://github.com/open-telemetry/opentelemetry-js/pull/2173/files#diff-417b84d68561f8d751332d48a1f9e52b83d6df08c0c19bc44826000ba7a8700eR40 ?

Reply:
Didn't read the PR body before reviewing files sorry :/

Reply:
@svrnm Well actually since you said you were done with the change, i think my comment stand

Reply:
Yes we should use the semconv pacakge.

Reply:
Of course, missed that one, there's so many of them ;-) 

like this @vmarchaud? -> https://github.com/open-telemetry/opentelemetry-js/pull/2173/commits/1b7edb509ba0ead921e4f5e005d70edf7dcc7c30

I am not 100% sure about the `PROCESS_RUNTIME_NAME` is this the right one for RUNTIME? And if so, would it also make sense to put a version there in the case of node? Or be more specific in the case of the browser?

Reply:
@dyladan @vmarchaud I had to apply another fix now to make things work, but I am not sure if this is really how the code should look like:

```
  /**
   * Returns a Resource that indentifies the SDK in use.
   */
  static createTelemetrySDKResource(): Resource {
    return new Resource({
      [SemanticResourceAttributes.TELEMETRY_SDK_LANGUAGE]:
        SDK_INFO[SemanticResourceAttributes.TELEMETRY_SDK_LANGUAGE],
      [SemanticResourceAttributes.TELEMETRY_SDK_NAME]:
        SDK_INFO[SemanticResourceAttributes.TELEMETRY_SDK_NAME],
      [SemanticResourceAttributes.TELEMETRY_SDK_VERSION]:
        SDK_INFO[SemanticResourceAttributes.TELEMETRY_SDK_VERSION],
    });
  }
```

Any thoughts?

Reply:
Yep thats perfect thanks !